server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -439,7 +439,7 @@ const orderMutations = {
 
         // Unpause contribution
         if (order.status === OrderStatuses.PAUSED) {
-          await order.unpause(req.remoteUser, { UserTokenId: req.userToken?.id });
+          await order.unPause(req.remoteUser, { UserTokenId: req.userToken?.id });
```

Comment:
There was a typo here causing "SUBSCRIPTION_RESUMED" activities to not be created